aws-rds-globalcluster/src/main/java/software/amazon/rds/globalcluster/BaseHandlerStd.java

```diff
@@ -199,7 +199,7 @@ protected ProgressEvent<ResourceModel, CallbackContext> createGlobalClusterWithS
             .done((describeDbClusterRequest, describeDbClusterResponse, proxyClient2, resourceModel, callbackContext) -> {
               final String arn = describeDbClusterResponse.dbClusters().get(0).dbClusterArn();
               try {
-                proxyClient2.injectCredentialsAndInvokeV2(Translator.createGlobalClusterRequest(resourceModel, arn), proxyClient2.client()::createGlobalCluster);
+                proxyClient2.injectCredentialsAndInvokeV2(Translator.createGlobalClusterRequest(resourceModel, arn, Tagging.TagSet.emptySet()), proxyClient2.client()::createGlobalCluster);
```

Comment:
Why are we unconditionally passing empty tagset here? Are we going to somehow derive tags from resourceModel down the line?